src/mixins/date-mixin.ts

```diff
@@ -19,7 +19,7 @@ export default class DateMixin extends Mixins(CommonMixin) {
    * @returns a promise to return a Date object
    */
   async getServerDate (): Promise<Date> {
-    const input = window.location.origin
+    const input = `${window.location.origin}/${process.env.VUE_APP_PATH}/`
```

Comment:
The previous endpoint worked locally but not when deployed. This one works (I tested earlier today). This just does a HEAD request by the way.